extensions/ql-vscode/src/view/results.tsx

```diff
@@ -113,7 +111,11 @@ class App extends React.Component<{}, ResultsViewState> {
             resultSet: {
               t: 'SarifResultSet',
               name: ALERTS_TABLE_NAME,
-              schema: { name: ALERTS_TABLE_NAME, version: 0, columns: [], tupleCount: 1 },
+              schema: {
```

Comment:
Is there a good reason for dropping the `version: 0`? Is that field just not in this type now and I am confused?